roles/nginxplus/tasks/main.yml

```
@@ -1,48 +1,53 @@
 ---
-- import_tasks: prerequisites/install-prerequisites.yml
+- name: nginxplus | check the pre-requisite software
+  ansible.builtin.import_tasks: prerequisites/install-prerequisites.yml
 
-- import_tasks: adjust_ulimit.yml
+- name: nginxplus | adjust limits
+  ansible.builtin.import_tasks: adjust_ulimit.yml
   when: nginx_type == "plus"
 
-- import_tasks: keys/apt-key.yml
+- name: nginxplus | add keys
+  ansible.builtin.import_tasks: keys/apt-key.yml
 
 - name: Install NGINX
   block:
 
-    - import_tasks: opensource/install-oss.yml
+    - ansible.builtin.import_tasks: opensource/install-oss.yml
       when: nginx_type == "opensource"
 
-    - import_tasks: plus/install-plus.yml
+    - ansible.builtin.import_tasks: plus/install-plus.yml
       when: nginx_type == "plus"
 
-    - import_tasks: conf/cleanup-config.yml
+    - ansible.builtin.import_tasks: conf/cleanup-config.yml
       when: nginx_cleanup_config
 
-    - import_tasks: modules/install-modules.yml
+    - ansible.builtin.import_tasks: modules/install-modules.yml
       when: true in nginx_modules.values()
 
-    - import_tasks: conf/upload-config.yml
+    - ansible.builtin.import_tasks: conf/upload-config.yml
       when: nginx_main_upload_enable
             or nginx_http_upload_enable
             or nginx_template_upload_enable
             or nginx_stream_upload_enable
             or nginx_html_upload_enable
             or nginx_ssl_upload_enable
 
-    - import_tasks: conf/template-config.yml
+    - ansible.builtin.import_tasks: conf/template-config.yml
       when: nginx_main_template_enable
             or nginx_http_template_enable
             or nginx_stream_template_enable
             or nginx_rest_api_enable
 
-    - import_tasks: conf/setup-status.yml
+    - ansible.builtin.import_tasks: conf/setup-status.yml
       when: nginx_status_enable
 
-    - import_tasks: conf/debug-output.yml
+    - ansible.builtin.import_tasks: conf/debug-output.yml
       when: nginx_debug_output
   when: nginx_enable
```

I don't see this set anywhere in the group_vars. Is it meant to be `nginx_something_enable`? Or is this a variable you set manually when the playbook/role is misbehaving? If that's the use case, I'd rename this to something like:
```suggestion
  when: nginx_debug
```